spid_cie_oidc/onboarding/onboardingforms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+
+
+class OnboardingRegistrationForm(forms.Form):
+    
+    organization_name = forms.CharField(
+        label= "organization Name"
+    )
+
+    url_entity = forms.URLField(
+        initial='https://',
+        label="url of the entity"
+    )
+
+    url_available = forms.URLField(
+        initial='http://',
+        label="url of the page where the SPID/CIE button is available"
```

Comment:
please all the texxt messages to the users must be localized, see gettext here
https://github.com/peppelinux/spid-cie-oidc-django/blob/3dcc0eca18d2e2dab2abcdeefb0c9a4b4ab66006/spid_cie_oidc/accounts/models.py#L4

then you can encapsulate all the string in `_( "string" )`